src/Main/AlexanderOnTest.NetCoreWebDriverFactory/WindowSize.cs

```diff
@@ -30,12 +33,61 @@ public enum WindowSize
         /// </summary>
         Fhd,
         /// <summary>
+        /// Quad HD (aka 1440p) - 2560 x 1440
+        /// </summary>
+        Qhd,
+        /// <summary>
+        /// Ultra HD-1 (aka 4k) - 3840 x 2160
+        /// </summary>
+        Uhd,
+        /// <summary>
```

Comment:
Would be safer to move these after the earlier non size values to maintain enum 'values' from v2.x

Reply:
done